.github/workflows/build-images-docs-builder.yaml

```diff
@@ -133,8 +133,8 @@ jobs:
         id: get_token
         uses: cilium/actions-app-token@61a6271ce92ba02f49bf81c755685d59fb25a59a # v0.21.1
         with:
-          APP_PEM: ${{ secrets.AUTO_COMMITTER_PEM }}
-          APP_ID: ${{ secrets.AUTO_COMMITTER_APP_ID }}
+          APP_PEM: ${{ secrets.AUTO_COMMITTER_PEM_202411 }}
+          APP_ID: ${{ secrets.AUTO_COMMITTER_APP_ID_202411 }}
```

Comment:
@aanm @ferozsalam it looks like we made this secret name change but did not expose the corresponding secret to the docs-builder environment.

Is it possible to switch the secret name back? Or do we need to create a new one (and presumably update the name since there's a date encoded here)? Do we need to be encoding a date into the secret name?

Reply:
Answer: We need to create a new one.